ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -494,19 +486,16 @@ const ControlledTierEraModule = ({profile, active, spinnerProps}): JSX.Element =
   const Module = ({profile}) => {
     return <FlexRow data-test-id={`module-${moduleName}`}>
       <FlexRow style={styles.moduleCTA}>
-        {((showRefresh && refreshAction)
-            ? <Refresh
-                refreshAction={refreshAction}
-                showSpinner={spinnerProps.showSpinner}/>
-            : null)}
+        {showRefresh && refreshAction
```

Comment:
nice.  I think the older style was a remnant of an earlier version that was more complex.